### ISSUE TITLE
higher framerates make cursor trail dissapear

### ISSUE THREAD

**Issue by waveneko** [itdelatrisu/opsu#424]
i'm probably the only person on earth that like the cursor trail but with my skin is looks really good. unfortunately, for the effect to be noticeable, my framerate needs to be lower than 120 (240 just about gives the effect but it's not as good). i typically unlock my framerate and my game runs at 1400fps on the beatmap selection screen. this high framerate gives me an advantage, but the trail is completely gone and it makes my cursor feel strange. the trail gives my mouse movement a satisfying fluidity to it. 

tl;dr, maybe give the trail intensity it's own slider to control how big the effect is, rather than it relying on framerate.

edit: ignore me. i just realised the trail isn't in the actual game. it's only on the menu.
